src/styles/blocks/day.scss

```diff
@@ -0,0 +1,24 @@
+.day {
```

Comment:
better to make the `day `a `calendar `element, because it is unlikely to be used without the `calendar `in the context of our project